jax/custom_derivatives.py

```diff
@@ -79,10 +79,18 @@ class custom_jvp:
   This class is meant to be used as a function decorator. Instances are
   callables that behave similarly to the underlying function to which the
   decorator was applied, except when a differentiation transformation (like
-  :py:func:`jax.jvp` or :py:func:`jax.grad`) is applied, in which case a custom user-supplied
-  JVP rule function is used instead of tracing into and performing automatic
-  differentiation of the underlying function's implementation. There is a single
-  instance method, ``defjvp``, which defines the custom JVP rule.
+  :py:func:`jax.jvp` or :py:func:`jax.grad`) is applied, in which case a custom
+  user-supplied JVP rule function is used instead of tracing into and
+  performing automatic differentiation of the underlying function's
+  implementation.
+
+  There are two instance methods available for defining the custom JVP rule:
+  :py:func:`~jax.custom_jvp.defjvp` for defining a *single* custom JVP rule for
```

Comment:
I am curious what is the role of "~" in the function reference?

Reply:
It’s documented [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax):

> If you prefix the content with ~, the link text will only be the last component of the target. For example, ``:py:meth:`~Queue.Queue.get` `` will refer to `Queue.Queue.get` but only display `get` as the link text.

I did that because it’s obvious that the links are to the class’s instance methods, so no need for the full name.